src/component/__snapshots__/pages.test.js.snap

```diff
@@ -0,0 +1,83 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Test home component Test home renderer 1`] = `
+<header
+  className="App-header"
+>
+  <nav
+    className="flex p-5 shadow-md"
+  >
+    <a
+      className="basis-1/3  text-white "
+      href="/home"
+      onClick={[Function]}
+    >
+      <svg
+        fill="currentColor"
+        height={28}
+        stroke="currentColor"
+        strokeWidth="0"
+        style={
+          Object {
+            "color": undefined,
+          }
+        }
+        viewBox="0 0 24 24"
+        width={28}
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M0 0h24v24H0z"
+          fill="none"
+        />
+        <path
+          d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
+        />
+      </svg>
+    </a>
+    <h3
+      className="basis-1/3 text-white flex justify-center font-bold"
+    >
+      Amazone Makeup
+    </h3>
+    <a
+      className="basis-1/3  text-white "
+      href="/setting"
+      onClick={[Function]}
+    >
+      <div
+        className="flex justify-end"
+      >
+        <svg
+          fill="currentColor"
+          height={28}
+          stroke="currentColor"
+          strokeWidth="0"
+          style={
+            Object {
+              "color": undefined,
+            }
+          }
+          viewBox="0 0 24 24"
+          width={28}
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M0 0h24v24H0V0z"
+            fill="none"
+          />
```

Comment:
- [ ] Please update your snapshots
![snap](https://user-images.githubusercontent.com/44745855/164894530-a44c45c6-2179-4b3b-86e9-31fc3f2db756.png)

Reply:
Noted and Fixed